src/formatting/formatter.rs

```diff
@@ -365,7 +378,7 @@ impl Formatter for EngFormatter {
 
                 let mut retval = match self.0.width as isize - digits {
                     isize::MIN..=0 => format!("{}", val.floor()),
-                    1 => format!(" {}", val.floor() as i64),
+                    1 => format!("{}", val.floor() as i64),
```

Comment:
See my previous comment.

Reply:
added the default `' '`

Reply:
I thought the point of this PR was to make this`' '` configurable... also I would prefer if this PR only affected the `.eng` formatter, not `.str`, unless you have a usecase for strings.